mlflow/protos/service.proto

```diff
@@ -406,6 +406,27 @@ service MlflowService {
     };
   }
 
+   // Log a param used for a run. A param is a key-value pair (string key,
```

Comment:
It is the comment for the logParam endpoint, it needs to be updated for the updateArtifactsLocation.

Reply:
Done